loadbalancer/algorithm.go

```diff
@@ -73,7 +73,7 @@ func fadeIn(now time.Time, duration time.Duration, exponent float64, detected ti
 func shiftWeighted(rnd *rand.Rand, ctx *routing.LBContext, now time.Time) routing.LBEndpoint {
 	var sum float64
 	rt := ctx.Route
-	ep := ctx.Route.LBEndpoints
```

Comment:
Please check that we do not accidentally use `ctx.Route.LBEndpoints` somewhere else in algorithms.